### ISSUE TITLE
Run live with specific image

### ISSUE THREAD

**Issue by orensbruli** [cseelye/docker-live#2]
Hello,
I would like to create a live USB that has dockers in it and that at startup would run a docker image. 
Do you think I could use your project to get this done? 
Any entry point to do it?

Thank you in advance.

**Comment by cseelye** [comment]
Yes, that is basically what I created this for myself.  You should be able to do this by creating a systemd service that starts the container, I just pushed a branch to demonstrate one method. https://github.com/cseelye/docker-live/tree/start-container - I added some info at the bottom of the readme to talk to it.
